frappe/oauth.py

```diff
@@ -363,9 +363,12 @@ def get_authorization_code_scopes(self, client_id, code, redirect_uri, request):
 
 	def get_jwt_bearer_token(self, token, token_handler, request):
 		now = datetime.datetime.now()
+		iat=round(now.timestamp())
+		
 		id_token = dict(
 			aud=token.client_id,
-			iat=round(now.timestamp()),
+			iat=iat,
+			exp=iat+token.expiry,
```

Comment:
should this be token.expires_in? - was reading bearer token docs and found expires_in everywhere 

for ex: https://oauthlib.readthedocs.io/en/latest/oauth2/tokens/bearer.html#generate-signed-jwt

Reply:
You're right don't know how i did that :facepalm: